packages/firebase_crashlytics/firebase_crashlytics/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.flutter.plugins.firebasecrashlyticsexample">
+    package="io.flutter.plugins.firebase.crashlytics.firebasecrashlyticsexample">
```

Comment:
I'm not sure if you meant to push this and the one above as the package on the file system is still: `example/android/app/src/main/java/io/flutter/plugins/firebasecrashlyticsexample/`

If we are changing this then the package should be renamed/moved to match. Would also prefer `io.flutter.plugins.firebase.crashlytics.example`

Reply:
Yeah, I prefer that name as well. I changed because it was the applicationId registered with Firebase and it is also the bundleId/applicationId for [iOS](https://github.com/FirebaseExtended/flutterfire/blob/@russell/flag-fatal-errors-crashlytics/packages/firebase_crashlytics/firebase_crashlytics/example/ios/GoogleService-Info.plist#L18). I'll swap it back.